### PR TITLE
Release v2.5.2

### PR DESCRIPTION
**Pull request by mikenikles**
Release PR

## Changes
- Add a version of the ES5 styles, ported to 3.x
- A number of fixes to support progressive web projects

## How To Test
- `npm link` this module
- run linting in a project that imports mobify-code-style

## TODOs:
- [x] +1
- [x] Updated README
- [x] Updated CHANGELOG
- [ ] Tag release (after merge)
- [ ] Publish to npm (after merge)

### CODE REVIEW

**Comment by DanielBrookRoberge** [comment]
I'm :+1: to get this out.

**Comment by marlowpayne** [comment]
My fix for #43 just made it in to `develop`! Please pull into this release. 🙏

**Comment by marlowpayne** [comment]
👍

**Comment by marlowpayne** [comment]
`package.json` version needs bumping. Looks like that didn't happen last time (for v2.5.1).

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Create a new pull request with the following settings:
+      * Base: `master`
+      * Compare: `release-vX.Y.Z`
+      Paste the contents of this checklist into this pull request.
+- [ ] Increment the version in `package.json`.
+- [ ] In the CHANGELOG.md file of this release branch, change the 'To be released' header to 'vX.Y.Z'.
+- [ ] Merge `release-vX.Y.Z` into `master` using the pull request (**don't delete the `release-vX.Y.Z` branch yet!!**).
+- [ ] [Publish to npm](https://docs.npmjs.com/cli/publish) from the release branch.
+- [ ] Draft a new Github release with the following settings:
+      * Tag version: `X.Y.Z` @ `release-vX.Y.Z`
```
Comment (marlowpayne) [inline RELEASE.md]: This seems to conflict with the docs from the [branching strategy repo](https://github.com/mobify/branching-strategy/blob/master/release-deployment.md#create-and-deploy-a-release), where it indicates that the tag target should be `master`, not the release branch.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      * Release title: `X.Y.Z - <island_name>`
```
Comment (marlowpayne) [inline RELEASE.md]: Also, I'm guessing this template was copy-pasted from Progressive App and Progressive Web. Not every repo we cut releases from needs an island name.

**Pull request merged**